packages/@aws-cdk/aws-rds/test/test.proxy.ts

```diff
@@ -138,6 +136,22 @@ export = {
       DBInstanceIdentifiers: ABSENT,
       TargetGroupName: 'default',
     }));
+    expect(stack).to(haveResourceLike('AWS::EC2::SecurityGroupIngress', {
```

Comment:
I would much prefer to see a separate test case for these. Something like 'db proxy configures the tcp security ingress'. This keeps test cases small and understandable, rather than a large set of 'expects' and fewer test cases.